### DISCUSSION TITLE
When clicking a symlink, that leads to nothing.. file manager just doesn't do anything.

### DISCUSSION THREAD

**Discussion by MrCheatEugene** [community/community#64308] [Repositories]
### Select Topic Area

Bug

### Body

Hi!

I'll be demonstrating the bug on [this repo](https://github.com/opengapps/opengapps/tree/master/sources), but I think it can happen in a lot of places also.
Here we have 5 links to folders. Well, they lead to nowhere if open in the new tab, but the issue here is not that - I see the issue in how it responds to that. No action, at all is performed on click. No exceptions, or popups like "Whoops, something went wrong". 

So I'm pretty sure this is a bug, in 2 places:
1. On pushing / uploading something happened, and now we have that link that goes to 404
2. And when it's uploaded, there is no sign to user that this folder doesn't exist for some reason.

Can you fix it?

**Comment by mutasim77** [comment]
Hey, this happened when I had a Git repository inside my Git repository. How did it happen? I cloned a repository (e.g., React) and then added it inside my own repository locally. Afterward, I tried to push that. While pushing, there wasn't any error. Then I checked my GitHub repository (the remote one), and the result was exactly the same (I couldn't click, etc.). So, what do those files contain?
